docs/src/index.md

```diff
@@ -2,28 +2,51 @@
 
 *🌊 Fast and friendly fluid dynamics on CPUs and GPUs.*
 
-Oceananigans.jl is a fast and friendly fluid flow solver written in Julia that can be run in 1-3 dimensions on CPUs
-and GPUs. It can simulate the incompressible Boussinesq equations, the shallow water equations, or the hydrostatic
-Boussinesq equations with a free surface. Oceananigans.jl comes with user-friendly features for simulating rotating
-stratified fluids including user-defined boundary conditions and forcing functions, arbitrary tracers, large eddy
-simulation turbulence closures, high-order advection schemes, immersed boundaries, Lagrangian particle tracking, and
-more!
-
-We strive for a user interface that makes Oceananigans.jl as friendly and intuitive to use as possible,
-allowing users to focus on the science. Internally, we have attempted to write the underlying algorithm
-so that the code runs as fast as possible for the configuration chosen by the user --- from simple
-two-dimensional setups to complex three-dimensional simulations --- and so that as much code
-as possible is shared between the different architectures, models, and grids.
-
-Oceananigans.jl is developed by the [Climate Modeling Alliance](https://clima.caltech.edu) and heroic external collaborators.
-
-## Getting help
-
-If you are interested in using Oceananigans.jl or are trying to figure out how to use it, please feel free to ask us
-questions and get in touch! If you're trying to set up a model then check out the examples and model setup
-documentation. Please feel free to [start a discussion](https://github.com/CliMA/Oceananigans.jl/discussions)
-if you have any questions, comments, suggestions, etc! There is also an #oceananigans channel on the
-[Julia Slack](https://julialang.org/slack/).
+Oceananigans is fast, friendly, and flexible software for finite volume simulations the nonhydrostatic
+and hydrostatic Boussinesq equations on CPUs and GPUs. Aside from its ability to run on GPUs (wow, fast!)
+Oceananigans receives it's powers from a powerful, intuitive, and hackable user interface that
+makes simple numerical experiments easy, and complex, creative numerical experiments possible.
+
+Oceananigans is written in Julia by the [Climate Modeling Alliance](https://clima.caltech.edu)
+and heroic external collaborators.
+
+## Quick install
+
+Oceananigans is a [registered Julia package](https://julialang.org/packages/), so the installation instructions are:
+
+1. [Download Julia 1.6](https://julialang.org/downloads/);
+
+2. Start up Julia and type
+
+```julia
+julia> using Pkg
+
+julia> Pkg.add("Oceananigans")
```

Comment:
```suggestion
julia> using Pkg; Pkg.add("Oceananigans")
```

Reply:
just a suggestion -- either way really!

Reply:
I was thinking the two line version is easier to read, but the one line version is nice and snappy

Reply:
leave it with 2 lines!